.eslintrc.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  "parser": "babel-eslint",
+  "rules": {
+    "strict": 0,
+    "no-console":0
```

Comment:
Because I wanted to keep my console.log lines to indicate if the user hasn't specified a route. I read into no-console and the concept is just that production level code shouldn't have console.logs at all... but I can remove them if you would prefer.